CHANGELOG.md

```diff
@@ -215,6 +215,22 @@ If we missed any change or there's something you'd like to discuss about this ve
 - No default values are set from Rspotify now, they will be left to the Spotify API.
 - ([#202](https://github.com/ramsayleung/rspotify/pull/202)) Add a `collaborative` parameter to `user_playlist_create`.
 - ([#202](https://github.com/ramsayleung/rspotify/pull/202)) Add a `uris` parameter to `playlist_reorder_tracks`.
+- ([]()) Update the endpoint signatures to pass parameters by reference, affected endpoint list:
```

Comment:
don't forget to update this

Reply:
fixed: [`df8d58a` (#206)](https://github.com/ramsayleung/rspotify/pull/206/commits/df8d58ada15e77e1612872c1dc63db011a653e37)